lib/lint-helpers.js

```diff
@@ -11,9 +11,13 @@ exports.isUsingAsar = () => {
 
 exports.isListeningForCrashEvents = () => {
   return Eval.execute(() => {
-    const webContents = require('electron').remote.getCurrentWebContents()
-    // Electron has a crashed listener, so look for more than 1
-    return webContents.listenerCount('crashed') > 1
+    const remote = require('electron').remote
+    const webContents = remote.getCurrentWebContents()
+    // For versions less than 1.x.y
+    // Electron has an crashed listener, so look for more than 1
+    const crashedForwarding = /^0/.test(remote.process.versions.electron)
```

Comment:
You don't need the `remote.` here, `process.versions.electron` is available directly in the renderer process.